node/resolvers/catalog/assemblyOption.ts

```diff
@@ -0,0 +1,65 @@
+export const resolvers = {
+  AssemblyOption: {
+    inputValues: async (
+      { inputValues }: AssemblyOption,
+    ) => {
+      return parseInputValues(inputValues)
```

Comment:
I see no point in separating and creating this function instead of writing the code here, but no prob.